data-raw/simulate-data.R

```diff
@@ -141,16 +141,17 @@ create_fake_date <- function(n, from = "1977-01-01", to = lubridate::today()) {
     sample(n, replace = TRUE)
 }
 
-#' Create a vector of random zero-padded integers.
+#' Create a vector of reproducible, random zero-padded integers.
 #'
 #' @param n The number of integers to generate.
-#' @param length An integer determining the length of the padded integer.
+#' @param length An integer determining the length of the padded integer. Generated integers of the same length are identical to facilitate joining by pnr, cpr, recnum and dw_ek_kontakt.
```

Comment:
The added text could be moved up into the roxygen description section (as a new paragraph below the title).